cstring.h

```diff
@@ -910,5 +912,47 @@ void stringDestroy(TString *s) {
     *s = (TString) {0};
 }
 
+bool charIsSep(char cur_ch){
+    char sep[5] = {' ', '\t', '\n', ',', '.'};
+    for (int i=0; i<5; i++){
```

Comment:
```suggestion
    char sep[] = {' ', '\t', '\n', ',', '.'};
    size_t len = sizeof(sep) / sizeof(*sep)
    for (int i=0; i < len; i++){
```
1. Можно создать массив, не указывая длину, а затем используя `sizeof` подсчитать её. Это полезно, так как в случае если захотим добавить новый символ в массив разделителей, то придётся только дописать его в конец, ничего больше не изменяя в коде
2. Лучше объявить массив `static const char` чтобы подчеркнуть, что это некая константа и помочь компилятору в оптимизации кода